pyrene/src/components/SingleSelect/SingleSelect.tsx

```diff
@@ -169,94 +172,52 @@ const SingleSelect = <ValueType extends unknown = DefaultValueType>({
   invalidLabel = '',
   title = '',
   value,
-  onChange = () => null,
+  onChange,
   onBlur = () => null,
   onFocus = () => null,
 }: SingleSelectProps<ValueType>): React.ReactElement => {
 
   const optionsObj = getOptionsObj(options, groupedOptions, sorted);
 
+  const props = {
```

Comment:
Maybe we should call this `selectProps` (or something else)? Normally, I think `props` is reserved for the component props itself that are passed in as a function argument to not cause confusion when used later in the file.